ext/swiftlint/Rakefile

```diff
@@ -10,16 +10,13 @@ namespace :swiftlint do
     REPO = 'https://github.com/realm/SwiftLint'
     VERSION = ENV['SWIFTLINT_VERSION'] || DangerSwiftlint::SWIFTLINT_VERSION
```

Comment:
What if a developer specifies another `SWIFTLINT_VERSION`? 
Let's say they want to use 0.42.0, in that case your validation will fail because `DangerSwiftlint::SWIFTLINT_HASH` will always give you the one for the hardcoded version.

A solution would be to only run the validation if `ENV['SWIFTLINT_VERSION']` is not specified.

Reply:
Oh good catch!

Reply:
Good catch, will update later today along with @ashfurrow observation. 👍🏻

Reply:
Good catch, will update later today along with @ashfurrow observation. 👍🏻